code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -394,6 +394,10 @@
 	var/fortify = 0
 	var/crest_defense = 0
 
+	// Baneling vars
+	/// Respawn charges, each charge makes respawn take 30 seconds. Maximum of 2 charges. If there is no charge the respawn takes 120 seconds.
+	var/stored_charge = 0
```

Comment:
Why on the mob? It was fine on the pod.

Reply:
Because of how we spawn the pod in

Reply:
Because now we just delete the pod everytime we spawn a new one in, so we wanna keep track of the charges in the ability instead of the pod

Reply:
But this's on the xeno, not in the ability. I'm being nitpicky though, this's fine.